dockerfiles/Dockerfile.openvino

```diff
@@ -57,9 +56,13 @@ WORKDIR /code
 ENV PATH /opt/miniconda/bin:/code/cmake-3.14.3-Linux-x86_64/bin:$PATH
 
 RUN git clone --recursive -b $ONNXRUNTIME_BRANCH $ONNXRUNTIME_REPO /onnxruntime && \
-    /bin/sh onnxruntime/dockerfiles/scripts/install_common_deps.sh && \
+    /bin/sh /onnxruntime/dockerfiles/scripts/install_common_deps.sh && \
     cd /onnxruntime/cmake/external/onnx && python3 setup.py install && \
     cp /onnxruntime/dockerfiles/LICENSE-IMAGE.txt /code/LICENSE-IMAGE.txt && \
+    cp /onnxruntime/dockerfiles/EULA.txt /code/EULA.txt && \
```

Comment:
Remove. The EULA.txt is already present in the image from L19 in /opt/intel/openvino_2019.1.144/licensing/EULA.txt ./openvino_2019.1.144/licensing/EULA.txt

Reply:
why does it need to be copied if it's already installed and in the image?